app/gui/language/ast/impl/src/macros/skip_and_freeze.rs

```diff
@@ -0,0 +1,144 @@
+//! Helper functions and data structures for [`SKIP`] and [`FREEZE`] macros support.
+//!
+//! [`SKIP`] and [`FREEZE`] macros change the behavior of the expression they are prepending.
+//!
+//! When used together in a single expression, [`SKIP`] macro always takes the priority and is
+//! placed at the beginning. For example, `SKIP FREEZE foo`.
+
+use enso_prelude::*;
+
+use crate::known;
+use crate::Ast;
+use crate::Crumbable;
+use crate::HasRepr;
+
+
+
+// =================
+// === Constants ===
+// =================
+
+/// The text representation of the [`SKIP`] macro's name.
+pub const SKIP_MACRO_IDENTIFIER: &str = "skip";
+/// The text representation of the [`FREEZE`] macro's name.
+pub const FREEZE_MACRO_IDENTIFIER: &str = "freeze";
```

Comment:
Let's change these to `SKIP` and `FREEZE` respectively, please. The new parser will handle them written this way.

Reply:
Changed